examples/bcs_mean_field.py

```diff
@@ -144,7 +144,7 @@ def main():
     n_site = 4
     # Number of fermions
     n_fermi = 4
-    # Hopping strength between neighboring sites
```

Comment:
This is correct in [American English](https://sapling.ai/usage/neighboring-vs-neighbouring)